README.md

```diff
@@ -7,11 +7,16 @@
 
 ## Usage
 
+### 0. Clone this repository
+```
+$ git clone git@github.com:bayesimpact/tds.git
```

Comment:
Consider removing the $ and setting the language for the code block to sh instead, so it's easier to copy and paste this line.